zypper-docker-build.sh

```diff
@@ -55,3 +55,10 @@ zypper --non-interactive al csm-ssh-keys
 # Remove all repos & scrub the zypper directory 
 zypper --non-interactive rr --all
 rm -f /etc/zypp/repos.d/*
+# Manually set the links that SLES neglects to do for us
+update-alternatives --install /usr/bin/easy_install easy_install /usr/bin/easy_install-3.9 99
+update-alternatives --install /usr/bin/pip pip /usr/bin/pip3.9 99
+update-alternatives --install /usr/bin/pip3 pip3 /usr/bin/pip3.9 99
+update-alternatives --install /usr/bin/pydoc3 pydoc3 /usr/bin/pydoc3.9 99
+update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 99
```

Comment:
It was Python 3.10 where the dependency issues started to eat us alive, so I think this is okay.